staging/src/k8s.io/apiserver/pkg/util/flag/ciphersuites_flag.go

```diff
@@ -48,6 +50,15 @@ var ciphers = map[string]uint16{
 	"TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305":  tls.TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305,
 }
 
+func TLSCipherPossibleValues() string {
+	cipherKeys := reflect.ValueOf(ciphers).MapKeys()
```

Comment:
Prefer gathering this without reflection. Iterate, insert into a `sets.NewString()`, and return the result of calling List() on the string set. That returns a []string and lets the caller decide how to render the list

Reply:
While you're in here, can you do the same for the TLSMinVersion flags?